Compile WARNING: commit e6397a98

The following compile warnings happen when compiling commit e6397a98 on Debian 10:

In file included from common/lc_pagesetupdialog.cpp:2:
common/lc_pagesetupdialog.h: In constructor ‘lcPageSetupDialog::lcPageSetupDialog(QWidget*, lcInstructionsPageSetup*)’:
common/lc_pagesetupdialog.h:21:25: warning: ‘lcPageSetupDialog::ui’ will be initialized after [-Wreorder]
  Ui::lcPageSetupDialog *ui;
                         ^~

common/lc_pagesetupdialog.h:20:27: warning:   ‘lcInstructionsPageSetup* lcPageSetupDialog::mPageSetup’ [-Wreorder]
  lcInstructionsPageSetup* mPageSetup;
                           ^~~~~~~~~~

common/lc_pagesetupdialog.cpp:6:1: warning:   when initialized here [-Wreorder]
 lcPageSetupDialog::lcPageSetupDialog(QWidget* Parent, lcInstructionsPageSetup* PageSetup)

--- a/common/lc_pagesetupdialog.h
+++ b/common/lc_pagesetupdialog.h
@@ -17,6 +17,6 @@ public slots:
 	void accept() override;
 
 private:
-	lcInstructionsPageSetup* mPageSetup;
 	Ui::lcPageSetupDialog *ui;
+	lcInstructionsPageSetup* mPageSetup;
 };

